src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../../common/variables';
+
+$activityThreadReplies-bar-wdth: 4px;
+
+.bcs-ActivityThread-button {
+    margin-left: $sidebarActivityFeedSpacingHorizontal;
+    color: $bdl-box-blue;
+    font-weight: bold;
+
+    &:hover {
+        margin-left: $sidebarActivityFeedSpacingHorizontal;
+        font-weight: bold;
+    }
+}
+
+.bcs-ActivityThread-loading {
+    padding-bottom: $bdl-grid-unit * 5;
+}
+
+.bcs-ActivityThreadReplies {
```

Comment:
Is this a separate component? If so, can we move these styles to a separate stylesheet?